packages/rn-tester-e2e/jest.config.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+module.exports = {
```

Comment:
We will also want `maxWorkers: 0` as Jest will normally spawn separate workers in parallel per suite.

Reply:
Hi @kelset, 
This is fixed in the last MR. Could you please resolve this comment?